mise.toml

```diff
@@ -0,0 +1,29 @@
+[tools]
+# Project overrides (formerly .pulumi.version)
+pulumi = '3.171'
+
+# See toolVersions at
+# https://github.com/pulumi/ci-mgmt/blob/master/provider-ci/internal/pkg/templates/defaults.config.yaml
+dotnet = '8'
+java = 'corretto-11.0.27'
+gradle = '7.6'
+nodejs = '20'
+python = '3.11'
+
+# See toolVersions at
+# https://github.com/pulumi/pulumi-kubernetes/blob/master/.ci-mgmt.yaml
+go = '1.24'
+
+# CONTRIBUTING.md depencies
+kubectl = '1.33.2'
+# TODO Find why we cannot just use npm and remove one extra dependency
```

Comment:
Probably worth taking this note out of here. Perhaps open an issue in ci-mgmt for moving from yarn to npm in all repos?

Reply:
https://github.com/pulumi/ci-mgmt/issues/1624

Reply:
(and removed from the code)